php/getfilepath.php

```diff
@@ -2,26 +2,67 @@
 $class = $argv[1];
 $file = $argv[2];
 $rootpath = $argv[3];
-$autoloader = $rootpath . '/vendor/autoload.php';
-if (is_readable($autoloader)) {
-    $loader = require $autoloader;
-
-    require __DIR__ . '/FQCN.php';
-
-    $contents = file_get_contents($file);
-    $fqns = new Hkt\FQCN();
-    $useClasses = $fqns->getAllUseStatements($contents);
-    if (array_key_exists($class, $useClasses)) {
-        echo $loader->findFile($useClasses[$class]);
-    } else {
-        // Get current namespace
-        $namespace = $fqns->getNamespace();
-        if ($namespace) {
-            $class = $namespace . '\\' . $class;
-        }
-        echo $loader->findFile($class);
-    }
 
+require __DIR__ . '/FQCN.php';
+
+$foundAutoLoader = false;
+
+// First try default location
+if(is_readable($autoloader)) {
```

Comment:
Oops, you are right!

The default location should be used here, but was deleted by accident.

I have updated the PR